packages/lib/services/noteList/defaultListRenderer.ts

```diff
@@ -57,7 +58,7 @@ const renderer: ListRenderer = {
 			background-color: var(--joplin-selected-color);
 		}
 
-		&:hover, :focus-visible > & > .content {
+		&:hover, &.-focus-visible > .content {
```

Comment:
Focus is no longer directly on each list item. Instead, the note list itself has focus and communicates the selected item using `aria-selected` and [`aria-activedescendant`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-activedescendant). MDN includes an [example](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role#example_1_a_single_select_listbox_that_uses_aria-activedescendant) that takes a similar approach.